server/index.js

```diff
@@ -18,6 +18,32 @@ const db = new pg.Pool({
   connectionString: process.env.DATABASE_URL
 });
 
+app.get('/api/home/project', (req, res, next) => {
```

Comment:
The convention to follow is "RESTful". You should name your collection routes in plural `/api/projects`.

https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/